experimental/coref/README.md

```diff
@@ -0,0 +1,108 @@
+
+```
+spacy project run prep-conll-data
+```
+
+After that you can execute `spacy project run all`.
+
+If you already have CoNLL 2012 data prepared and concatenated into one file per split, you can specify the paths to the training, dev, and test files directly in `project.yml`, see the `vars` section. After doing so you can run `spacy project run all`.
+
+## Using the Trained Pipeline
+
+After training the pipeline (or downloading a pretrained version), you can load and use it like this:
+
+```
+import spacy
+nlp = spacy.load("training/coref")
+
+doc = nlp("John Smith called from New York, he says it's raining in the city.")
+# check the word clusters
+print("=== word clusters ===")
+word_clusters = [val for key, val in doc.spans.items() if key.startswith("coref_head")]
+for cluster in word_clusters:
+    print(cluster)
+# check the expanded clusters
+print("=== full clusters ===")
+full_clusters = [val for key, val in doc.spans.items() if key.startswith("coref_cluster")]
+for cluster in full_clusters:
+    print(cluster)
+```
+
+The prefixes used here are a user setting, so you can customize them for your
+own pipelines.
+
+
+## 📋 project.yml
+
+The [`project.yml`](project.yml) defines the data assets required by the
+project, as well as the available commands and workflows. For details, see the
+[spaCy projects documentation](https://spacy.io/usage/projects).
+
+### ⏯ Commands
+
+The following commands are defined by the project. They
+can be executed using [`spacy project run [name]`](https://spacy.io/api/cli#project-run).
+Commands are only re-run if their inputs have changed.
+
+| Command | Description |
+| --- | --- |
+| `prep-ontonotes-data` | Rehydrate the data using OntoNotes |
+| `prep-test-data` | Prepare minimal dataset for CI testing. Note this will overwrite train/dev/test data! |
+| `preprocess` | Convert the data to spaCy's format |
+| `train-cluster` | Train the clustering component |
+| `prep-span-data` | Prepare data for the span resolver component. |
+| `train-span-resolver` | Train the span resolver component. |
+| `assemble` | Assemble all parts into a complete coref pipeline. |
+| `eval` | Evaluate model on the test set. |
+
+### ⏭ Workflows
+
+The following workflows are defined by the project. They
+can be executed using [`spacy project run [name]`](https://spacy.io/api/cli#project-run)
+and will run the specified commands in order. Commands are only re-run if their
+inputs have changed.
+
+| Workflow | Steps |
+| --- | --- |
+| `prep` | `preprocess` |
+| `train` | `train-cluster` &rarr; `prep-span-data` &rarr; `train-span-resolver` &rarr; `assemble` |
+| `ci-test` | `prep-test-data` &rarr; `train-cluster` &rarr; `prep-span-data` &rarr; `train-span-resolver` &rarr; `assemble` &rarr; `eval` |
```

Comment:
I would consider removing the `ci-test` workflow entirely and instead call the separate steps from within the unit test. Just so that this doesn't show up for users and doesn't potentially confuse them as to which workflow is most appropriate to them. I don't think we'd typically run this `ci-test` workflow from the commandline ourselves, right?

Reply:
That's a fair point. Since it's designed to be as similar to the real thing as possible it felt right to use the project file, but keeping it out of the way for users is more important.